.pre-commit-config.yaml

```diff
@@ -1,8 +1,8 @@
 default_language_version:
-  python: python3.12
+  python: python3.11
 repos:
```

Comment:
What is the reason for this change?

Reply:
had no 3.12 locally, commit was unintentional. removed now